packages/playground/src/pages/pipeline-edit-page.tsx

```diff
@@ -49,7 +49,14 @@ MonacoGlobals.set({
 })
 
 function GraphView() {
-  const nodes = useMemo(() => getNodesFromPipelineYaml(pipeline as unknown as string), [])
+  const [nodes, setNodes] = useState<Node[]>(getNodesFromPipelineYaml(pipeline))
+
+  useEffect(() => {
```

Comment:
this looks weird. can we initialise the state with an empty array instead?

Reply:
Removed it, we don't need this line. Was testing something earlier.